src/io/graphics.rs

```diff
@@ -114,14 +114,8 @@ impl Editor {
                          18,
                          Color::rgba(74, 74, 74, 255));
 
-        status_bar(self, self.status_bar.mode.to_owned(), 0, 4);
-        let sb_file = self.status_bar.file.clone();
-        status_bar(self, sb_file, 1, 4);
-        let sb_cmd = self.status_bar.cmd.clone();
-        status_bar(self, sb_cmd, 2, 4);
-        let sb_msg = self.status_bar.msg.clone();
-        status_bar(self, sb_msg, 3, 4);
-
+        status_bar(self);
```

Comment:
I have updated the method to become an editor instance method and I have changed the name to draw_status_bar as requested.